deployment/cdk/opensearch-service-migration/options.md

```diff
@@ -24,6 +24,7 @@ These tables list all CDK context configuration values a user can specify for th
 | trafficReplayerGroupId               | string  | "logging-group-default"                                                | The Kafka consumer group ID the Replayer will use, if not specified a default ID will be used                                                                                                                                                                                                    |
 | trafficReplayerUserAgentSuffix       | string  | "AWS/test/v1.0.0"                                                      | A custom user agent that will be provided to the Replayer using the `--user-agent` flag. This will append the provided user agent to any existing user agents when requests are made to the target cluster. This setting could also be specified with the `trafficReplayerExtraArgs` option      |
 | trafficReplayerExtraArgs             | string  | "--sigv4-auth-header-service-region es,us-east-1 --speedup-factor 5"   | Extra arguments to provide to the Replayer command. This includes auth header options and other parameters supported by the [Traffic Replayer](../../../TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java).                                     |
+| trafficReplayerMaxUptime          | string  | "PT2H"                                                                 | The maximum uptime for the Traffic Replayer service, specified in ISO 8601 duration format. This controls how long the Traffic Replayer will run before automatically shutting down. Example values: "PT30M" (30 minutes), "PT2H" (2 hours). When this duration is reached, ECS will initiate the startup of a new Traffic Replayer task to ensure continuous operation. This mechanism ensures that the Traffic Replayer service can manage its resources effectively and prevent issues associated with long running processes. Set to the greater of the given value 5 minutes. |
```

Comment:
2H seems a little arbitrary.  I would say in terms of most easy to understand, hourly, daily (if it's a big problem to recycle too often) or even every 6 hours.  Doing this often enough that an operator WILL see it happen could be a plus.

Where is the default actually defined?

Reply:
There is no default currently so the deployment will not deploy the artifacts (maxUptimeContainer) needed to monitor this. 

I'll change this to discuss hourly and daily as examples